### PR TITLE
Extract table created if it doesn't exist, and simple extract wrote

### PR DESCRIPTION
**Pull request by MichaelCullenMadeTech**

### CODE REVIEW

File: extract_data.py [hunk 1]
```diff
@@ -16,6 +15,41 @@ def create_db_connection(database):
...
+                               PRIMARY KEY(uuid, audit_timestamp, type),
+                               CONSTRAINT audit_event_uuid_idempotent UNIQUE(uuid, audit_timestamp, type)
+                           ) PARTITION BY RANGE(audit_timestamp)
+                """)
+
+
+def extract_data():
+    try:
+        with create_db_connection(REPLICA_DATABASE) as replica_connection, create_db_connection(TRANSFORM_DATABASE) as transform_connection:
+            with replica_connection.cursor().copy(f"COPY {replica_schema}.audit_event TO STDOUT (FORMAT BINARY)") as copy_replica:
```
Comment (RhysJones125) [inline extract_data.py:45]: Replica Schema doesn't exist within this function

File: extract_data.py [hunk 2]
```diff
@@ -16,6 +15,41 @@ def create_db_connection(database):
...
+                with transform_connection.cursor().copy(f"COPY {transform_schema}.audit_event FROM STDIN (FORMAT BINARY)") as copy_transform:
```
Comment (RhysJones125) [inline extract_data.py:46]: I don't think there is a transform schema specified in any of the environment variables. Do we want to create one?

**Comment by RhysJones125** [comment]
Would it be worth also spin up 2 databases locally as to avoid the need of deploying our work to test it?

File: extract_data.py (hunk 2 above)
Comment (MichaelCullenMadeTech) [reply extract_data.py:46]: Probably not actually, let me remove that then.

**Pull request merged**